feature.py

```diff
@@ -67,57 +67,75 @@ def forward(self, x: th.Tensor) -> th.Tensor:
 class Residual(nn.Module):
     """CBA + Residual Block."""
 
-    def __init__(self, base: nn.Module):
-        self.base = base
+    def __init__(
+        self, 
+        c_in: int, 
+        c_out: int, 
+        padding: Union[int, Tuple[int, int]] = 1, 
+        stride: Union[int, Tuple[int, int]] = 1
+    ):
+        super().__init__()
+        self.conv1 = CBA(c_in, c_out, padding=padding, stride=stride)
+        self.conv2 = CBA(c_out, c_out, lrelu_eps=None)
+        self.residual = nn.Sequential()
+        if c_in != c_out or stride != 1:
```

Comment:
Why check for `if c_in != c_out or stride != 1` rather than `if use_residual`?

Reply:
I thought about introducing a new variable like use_residual, but somehow I preferred not to create a new one...